steps/get.ts

```diff
@@ -81,6 +81,9 @@ const fetchLanguages = async (): Promise<void> => {
   rows.forEach((item) => {
     const url = $(item).find('td.list-highlight-background:first-child a').attr('href')
     const slug = /locale=(.*)$/.exec(url)?.pop()
+    if (argv.withoutLanguageVariants && slug.includes('_')) {
```

Comment:
It should not be based on the slug, but on the lang. This is not robust.

Reply:
Slug it just a variable name. It is filled from the `locale` part of the URL.
All other steps and functions use the value from this variable to define the language.